Comp2025/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -286,7 +298,7 @@ private void initDashboard( )
 
     // Add commands
     SmartDashboard.putData("ClRunStowed", getMoveToPositionCommand(this::getElevatorStowed));
-    SmartDashboard.putData("ClRunCoralL1", getMoveToPositionCommand(this::getElevatorCoralL1));
+    SmartDashboard.putData("ClRunCoralL1", getMoveToPositionCommand(this::getHeightCoralL1));
```

Comment:
I suggest putting in a few more of these SmartDashboard buttons for testing right now to achieve more of the available heights. It will be easier to debug.